rules/config

```diff
@@ -30,7 +30,16 @@ DEFAULT_BUILD_LOG_TIMESTAMP = none
 
 # SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD - use native dockerd for build.
 # If set to y SONiC build container will use native dockerd instead of dind for faster build
+
+# SONIC_CONFIG_NATIVE_DOCKERD_SHARED - native dockerd for build runs on a shared build server.
+# Note: Enabling this option automatically sets SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD=y.
+# If both options are set, this one prevails.
+# If set to y, special handling of the docker image file names is needed to avoid conflicts with
+# other SONiC build jobs on the same server. This requires changes to the Dockerfile.j2 FROM statement
+# in the dockers/ and platform/ subdirs to use a variable reference instead of an explicit image name.
+# The following platforms support this option: broadcom, vs
```

Comment:
why do we have this limitation? it should be a generic solution.

Reply:
It is a generic solution. Other platforms will be supported.

Reply:
can you remove the comments then

Reply:
Removed it.